### ISSUE TITLE
[MHV Medications / Accessibility] Focus skips over sort feature after applying filters

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#125140]
### 📝 User story

As a keyboard user, I want focus to move through content in the same order as the visual order, so that I can predict where focus will go next when I select the Tab key.

As a screen reader user, I want systems that move focus not to skip past relevant content and functionality, so that I can be sure not to miss anything related to the task at hand.

### 📝 Problem statement

Selecting the Apply Filter button skips over the sort feature and moves focus to the results details.

### 🌌 Supporting screenshots or videos

https://github.com/user-attachments/assets/3b12aec4-86c3-4198-82a6-773ff580ec5a

This video demonstrates keyboard navigation and focus management. The arrow key moves focus between radio buttons, the Tab key moves focus to the Apply Filters button, the Enter key applies the filter to the results list and moves focus automatically to the results details, Showing x or x results…, skipping over the Sort medications feature.

### 👣 Reproduction steps

1. Navigate to the [Medications list page](https://staging.va.gov/my-health/medications?page=1) (username: [karsyn.gpsysasynctwo@id.me](mailto:karsyn.gpsysasynctwo@id.me)).
2. Use the Tab key to focus on the Filters accordion; select Enter to open the accordion panel.
3. Use the arrow key to select a radio button option.
4. Use the Tab key to move focus to the Apply Filters button; select Enter to apply filters.
6. Notice the focus indicator moves automatically to the "Showing x of x" results details.

### 🔗 Point of contact

@sarahhortonadhoc

### ⚠️ Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

#### Accessibility Standards

- [Guideline 2.4 Navigable: Provide ways to help users navigate, find content, and determine where they are](https://www.w3.org/WAI/WCAG21/Understanding/navigable)
  - [SC 2.4.3 Focus Order (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)

#### Experience Standards
- [Consistency: User has a predictable and unchanging experience across all VA.gov and VA Mobile App services, tools, and pages](https://design.va.gov/about/experience-standards/#category-consistency)
  - [User experiences predictable interactions with components and patterns](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components-and-patterns)

### 𝌣 Proposed solution or next steps

Revisit the filter and sort design and implementation.

### 𝌣 Supporting resources

- [Focus management](https://design.va.gov/accessibility/focus-management) — from VADS

### ✅ Acceptance criteria

- [ ] The system does not skip over content and functionality in its focus management.
- [ ] The keyboard user can accurately predict which element will receive focus.
- [ ] The screen reader does not miss relevant content and functionality.

### ✅ Possible tasks

- [ ] Review, refine, and acknowledge feedback.
- [ ] Update ticket information (title, assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.

**Comment by sarahhortonadhoc** [comment]
@ckeif-iat @michaelbrodsky-a6 Making sure you saw this update. This issue is marked done/not planned but we can address this issue (see the previous comment). We can improve the accessibility for this specific issue by implementing this targeted update.
